tools/flowc/backends/nim/natives/string/toUpperCase.nim

```diff
@@ -1,2 +1,22 @@
-proc $F_0(toUpperCase)*(s: string): string =
-  return unicode.toUpper(s)
+proc $F_0(toUpperCase)*(s: String): String =
```

Comment:
This will only convert English letters to Upper case. Maybe just convert to native strings and then use the native toUpper() function, and convert that back to utf16

Reply:
Ok, switched to unicode.toUpper/toLower wrappers